vueui/src/components/view/security/guntracker.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <h3>Firearm Control System</h3>
+    <template v-if="screen">
+      <vui-button :params="{ lock: 1 }">Unlock Console</vui-button>
+    </template>
+    <template v-else>
+      <vui-button :params="{ lock: 1 }">Lock Console</vui-button>
```

Comment:
It's better to make it send diffrent lock value depending on whenever you want it to unlock or lock console. If ui get out of sync then it should cause less confusion.

Reply:
This is no longer relevant now that it's not a console anymore.